ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Please try to adhere to the guidelines below as much as possible when submitting your issue.
+- Verify that your issue is not being currently addressed by other [open issues](https://github.com/scikit-learn/scikit-learn/issues) or [pull requests](https://github.com/scikit-learn/scikit-learn/pulls).
```

Comment:
I reckon we should include closed issues (and possibly closed PRs) and use a link like https://github.com/scikit-learn/scikit-learn/issues?q= so they can search through issues and PRs at once.